aspnetcore/mvc/views/tag-helpers/built-in/anchor-tag-helper.md

```diff
@@ -66,6 +65,33 @@ If no `asp-controller` attribute is specified, the default controller calling th
  
 If the attribute `asp-action` is `Index`, then no action is appended to the URL, leading to the default `Index` method being called. The action specified (or defaulted), must exist in the controller referenced in `asp-controller`.
 
+### asp-page
+
+Use the `asp-page` attribute in an anchor tag to set its URL to point to a specific page. Prefixing the page name with a forward slash "/" creates the URL. The URL in the sample below points to the "Speaker" page in the current directory.
```

Comment:
I believe `asp-page`  is for RP only - right? Need to say so. Might be good to link to source code.

Reply:
Good info, add that RP typically use "./" and then point to an example. Put a anchor on the page you point to.

Reply:
We've been directed to present RP first and as the primary UI dev approach.